### PR TITLE
Feature/40 add title for id page

### LINKED ISSUE #40: Add title for [id] page
As this page wasn't finished when the other titles were added the page for a single poll needs the title added. Also it could be dynamic, maybe displaying the poll id of the current poll.

### PR DESCRIPTION
**Pull request by NiklasBGit**
title is now displayed upon entering the page for a single poll the first time. On reload POLLme is displayed because I have no idea why it does not work.

### CODE REVIEW

File: pages/polls/[id].vue [hunk 1]
```diff
@@ -1,13 +1,45 @@
...
 
 </script>
 
+<script>
+export default {
+  beforeMount() {
+    this.fetchPoll().then((result) => {
+      this.poll = result;
+      console.log(result)
+      document.title = result.value ? result.value.name : "POLLme";
```
Comment (GrafBunsen) [inline pages/polls/[id].vue]:
```suggestion
document.title = 'POLLmelo - ' + (result.value ? result.value.name : "");
```
i added a POLLmelo infront, I think it is better this way

**Review by GrafBunsen — CHANGES_REQUESTED** [review]
there is a lot of code in this pr but only line 20 in pages/polls/[id].vue is important isn't it?

**Comment by NiklasBGit** [comment]
I suppose that might be the component for the poll item, which wasn't merged when I implemented the feature and I had to build on branch 26 to implement it?

**Comment by GrafBunsen** [comment]
> I suppose that might be the component for the poll item, which wasn't merged when I implemented the feature and I had to build on branch 26 to implement it?

might be. if we merge in the right order all the other code is redundant, so we wait to merge this untill #38 is merged

**Review by GrafBunsen — APPROVED** [review]

File: pages/polls/[id].vue [hunk 2]
```diff
@@ -39,6 +39,35 @@ export default {
 }
 </script>
 
+<script>
+export default {
+  beforeMount() {
+    this.fetchPoll().then((result) => {
+      this.poll = result;
+      console.log(result)
```
Comment (GrafBunsen) [inline pages/polls/[id].vue]:
```suggestion
```

**Review by GrafBunsen — CHANGES_REQUESTED** [review]

File: pages/polls/[id].vue [hunk 3]
```diff
@@ -39,6 +39,34 @@ export default {
...
+      const poll = await useFetch( () => `https://backberry.ddev.site/api/polls/${pollId}`, {
+        method: 'get',
+        initialCache: false,
+      });
+      return poll.data;
+    }
+  }
+}
+</script>
+
```
Comment (GrafBunsen) [inline pages/polls/[id].vue]:
```suggestion
```
remove duplicate code

**Review by GrafBunsen — CHANGES_REQUESTED** [review]

**Review by GrafBunsen — APPROVED** [review]

**Pull request merged**